Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 FROM golang:1.15-alpine AS build
 WORKDIR /work
-RUN apk --no-cache add build-base git gcc
+RUN apk --no-cache add build-base git gcc libseccomp-dev pkgconfig
```

Comment:
What is `pkgconfig` used for here? Should it be `pkg-config`?

Reply:
The package is called `pkgconfig`, but the resulting binary is `pkg-config`, yeah :grin:. Looks like that it is not necessary at all any more, removing it.